### ISSUE TITLE
Extract Interface from a Class

### ISSUE THREAD

**Issue by lievencardoen** [omecra/de.omecra.oedt.updatesite#84]
I seem to remember that Visual Studio has something like extracting/generating an interface from a Class. Seems like a great feature to add to OEDT.

https://stackoverflow.com/questions/9277011/generate-interface-from-existing-class

**Comment by lneumeier** [comment]
Yes, the hard part is the replacement of all occurrences, but yes, it's on my list.
